Update signing docs (#30296)
grafana/grafana: docs/sources/developers/plugins/sign-a-plugin.md

Plugins signed under the community or commercial signature level are considered _public plugins_. Public plugins are published on [Grafana Plugin](https://grafana.com/plugins). For more information about installing public plugins, refer to [Install Grafana plugins]({{< relref "../../plugins/installation.md" >}}).

> 1. Submit your plugin for review by creating a pull request on [grafana-plugin-repository](https://github.com/grafana/grafana-plugin-repository).

Reviewer: ```suggestion
1. Submit your plugin for review by creating a pull request on the [grafana-plugin-repository](https://github.com/grafana/grafana-plugin-repository).
```

Reviewer 2: Please change "on" to "in".

Author: ```suggestion
1. Submit your plugin for review by creating a pull request in the [grafana-plugin-repository](https://github.com/grafana/grafana-plugin-repository).
```

=GdNq
-----END PGP SIGNATURE-----
```

## Troubleshooting issues while signing your plugin

### Why am I getting a "Modified signature" in Grafana?

> Due to a issue when signing the plugin on Windows, grafana-toolkit generates an invalid MANIFEST.txt. You can fix this by opening the MANIFEST.txt and replacing all double backslashes, `\\`, with a forward slash, `/`. You need to do this every time you sign your plugin.

Reviewer: ```suggestion
Due to an issue when signing the plugin on Windows, grafana-toolkit generates an invalid MANIFEST.txt. You can fix this by replacing all double backslashes, `\\`, with a forward slash, `/` in the MANIFEST.txt file. You need to do this every time you sign your plugin.
```

### Error signing manifest: Field is required: rootUrls

> If you're trying to sign a **public** plugin, this means that your plugin doesn't yet have a plugin signature level assigned to it yet. A Grafana team member will assign a signature level to your plugin once it has been reviewed and approved. For more information, refer to [Sign a public plugin](#sign-a-public-plugin).

Reviewer: ```suggestion
If you're trying to sign a **public** plugin, this means that your plugin doesn't have a plugin signature level assigned to it yet. A Grafana team member will assign a signature level to your plugin once it has been reviewed and approved. For more information, refer to [Sign a public plugin](#sign-a-public-plugin).
```
